components/AppBar/src/MDCAppBarNavigationController.h

```diff
@@ -41,6 +41,37 @@
        willAddAppBarViewController:(nonnull MDCAppBarViewController *)appBarViewController
            asChildOfViewController:(nonnull UIViewController *)viewController;
 
+/**
+ Asks the receiver to determine the tracking scroll view for a given view controller.
+
+ If this method is not implemented, the app bar navigation controller will extract the first
+ UIScrollView instance in the view controller's view hierarchy using a depth-first view traversal.
+
+ Implement this method when you need to change the default behavior of the tracking scroll view
+ detection. The suggested tracking scroll view will contain the scroll view that would have been
+ used; you can ignore this value.
+
+ Your implementation will likely return the suggested tracking scroll view by default, with custom
+ logic being implemented for specific view controllers that require it.
+
+ @param navigationController The app bar navigation controller instance that owns the view
+ controller.
+ @param viewController The view controller for which the tracking scroll view should be determined.
+ @param scrollView A suggested tracking scroll view. This is the first UIScrollView instance
+ detected by a depth-first view traversal of @c viewController's view hierarchy. This suggestion
+ can be ignored.
+ @return The tracking scroll view to be used for this view controller. If nil is returned then no
+ tracking scroll view will be set.
+
+ @note This method will only be invoked if a new App Bar instance is about to be added to the view
+ controller. If a flexible header is already present in the view controller, this method will not
+ be invoked.
+ */
+- (nullable UIScrollView *)appBarNavigationController:
+                               (nonnull MDCAppBarNavigationController *)navigationController
+                  trackingScrollViewForViewController:(nonnull UIViewController *)viewController
+                          suggestedTrackingScrollView:(nullable UIScrollView *)scrollView;
```

Comment:
What is the use case for the `suggestedTrackingScrollView`? I would advocate removing this parameter unless we have a specific need for it and can demonstrate that need in an example.

My thought is that when the method is called one of two situations seems reasonable:
1.   The client inspects the provided VC and performs its own calculation to retrieve the scroll view for the view controller.
2.   The client inspects the provided VC and decides it doesn't want to override any default and returns `nil`.

Reply:
There may be legitimate use cases in which no tracking scroll view is desired, so treating nil as "use the default" will unfortunately remove the ability for a client to express this.

Reply:
Ack.